components/PaypalButtons.js

```diff
@@ -56,24 +54,32 @@ class PaypalButton extends React.Component {
     return actions.order.create({
       purchase_units: [
         {
-          description: +"Mercedes G-Wagon",
+          description: +'Mercedes G-Wagon',
```

Comment:
I believe this `+` before the string is a typo. It's only used to cast a string to a number